src/index.tsx

```diff
@@ -9,57 +9,50 @@ import {
   SearchBarProps,
 } from './types';
 
+import { screensEnabled } from './core';
```

Comment:
Could we even maybe remove `index.native.tsx` at all and just do proper platform checks so it all works correctly? Or is there something we cannot manage like this?